packages/lodestar-beacon-state-transition/src/fast/util/persistentVector.ts

```diff
@@ -0,0 +1,332 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+const BIT_WIDTH = 5;
```

Comment:
maybe move this to separate package in monorepo so it's easier to move to separate repository?